src/ServiceManagement/HDInsight/Commands.HDInsight/Cmdlet/NewAzureHDInsightClusterCmdlet.cs

```diff
@@ -47,6 +47,49 @@ public NewAzureHDInsightClusterCmdlet()
             this.command = ServiceLocator.Instance.Locate<IAzureHDInsightCommandFactory>().CreateCreate();
         }
 
+        /*
+         * Parameter Sets:
+         * 
+         * ParameterSetClusterByConfigWithSpecificSubscriptionCredentials
+         *      Position 00: Name
+         *      Position 01: Config
+         *      Position 02: Subscription
+         *      Position 03: Certificate
+         *      Position 04: Location
+         *      Position 05: Credential
+         *      Position 06: ???
+         *      Position 07: Endpoint
+         *      Position 08: HostedService
+         *      Position 09: Version
+         *      Position 10: OSType
+         *      Position 11: SshCredential
+         *      Position 11: SshPublicKey
+         *      
+         * ParameterSetClusterByNameWithSpecificSubscriptionCredentials
+         *      Position 00: Name
+         *      Position 01: Subscription
+         *      Position 02: Certificate
+         *      Position 03: Location
+         *      Position 04: DefaultStorageAccountName
+         *      Position 05: DefaultStorageAccountKey
+         *      Position 06: DefaultStorageContainerName
+         *      Position 07: Credential
+         *      Position 08: ???
```

Comment:
What does "???" mean? Please add comment if this position is skipped intentionally/by mistake.